packages/documentation/src/stories/get-started/intranet-header.stories.mdx

```diff
@@ -0,0 +1,48 @@
+import { Meta } from '@storybook/addon-docs';
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+
+<Meta
+  title="Get Started/Intranet-Header"
+  parameters={{
+    previewTabs: {
+      canvas: {
+        hidden: true,
+      },
+    },
+    viewMode: 'docs',
+    badges: [BADGE.NEEDS_REVISION],
```

Comment:
We still need to port the docs or provide a web component, so I'll add the todo label as well.

```suggestion
    badges: [BADGE.NEEDS_REVISION, BADGE.TODO],
```